synapse/util/retryutils.py

```diff
@@ -35,6 +35,29 @@ def __init__(self, retry_last_ts, retry_interval, destination):
 
 
 @defer.inlineCallbacks
+def would_retry(destination, clock, store):
```

Comment:
Of course another way to do this is just to call `get_retry_limiter`, and throw away the result, and see if you get an exception.